src/main/resources/default/assets/scripts/blob-upload-url-popover.js

```diff
@@ -0,0 +1,158 @@
+        body._inputField.value = '';
+    });
+
+    function buildContainer() {
+        const _container = document.createElement('div');
+        _container.classList.add('popover-content-max');
+        _triggerElement.parentElement.appendChild(_container);
+        return _container;
+    }
+
+    function buildBody() {
+        const body = {};
+        buildInputFormGroupCol(body);
+        buildOkButtonCol(body);
+        buildCancelButtonCol(body);
+        buildErrorCol(body);
+        buildBodyContainer(body);
+        return body;
+    }
+
+    function buildInputFormGroupCol(body) {
+        body._inputLabel = document.createElement('label');
+        body._inputLabel.innerHTML = i18n.label;
+
+        body._inputField = document.createElement('input');
+        body._inputField.classList.add('form-control');
+        body._inputField.setAttribute('type', 'text');
+        body._inputField.setAttribute('placeholder', 'https://');
+
+        body._formGroup = document.createElement('div');
+        body._formGroup.classList.add('form-group');
+        body._formGroup.append(body._inputLabel, body._inputField);
+
+        body._inputCol = document.createElement('div');
+        body._inputCol.classList.add('col', 'col-12');
+        body._inputCol.appendChild(body._formGroup);
+    }
+
+    function buildOkButtonCol(body) {
+        body._buttonOk = document.createElement('button');
+        body._buttonOk.classList.add('btn', 'btn-block', 'btn-primary', 'btn-apply');
+        body._buttonOk.innerHTML = i18n.ok + ' <i class="fa-solid fa-check"></i>';
+        body._buttonOk.addEventListener('click', function (event) {
+            event.preventDefault();
+            updateURL(body._inputField.value);
+        });
+
+        body._leftCol = document.createElement('div');
+        body._leftCol.classList.add('col', 'col-12', 'col-md-6');
+        body._leftCol.appendChild(body._buttonOk);
+    }
+
+    function buildCancelButtonCol(body) {
+        body._buttonCancel = document.createElement('div');
+        body._buttonCancel.classList.add('btn', 'btn-block', 'btn-close', 'btn-outline-secondary');
+        body._buttonCancel.innerHTML = i18n.cancel + ' <i class="fa-solid fa-close"></i>';
+        body._buttonCancel.addEventListener('click', function () {
+            $(_triggerElement).popover('toggle');
+        });
+        body._rightCol = document.createElement('div');
+        body._rightCol.classList.add('col', 'col-12', 'col-md-6');
+        body._rightCol.appendChild(body._buttonCancel);
+    }
+
+    function buildErrorCol(body) {
+        body._errorCol = document.createElement('div');
+        body._errorCol.classList.add('col', 'col-12');
+    }
+
+    function buildBodyContainer(body) {
+        body._container = document.createElement('div');
+        body._container.classList.add('row');
+        body._container.append(body._inputCol, body._errorCol, body._leftCol, body._rightCol);
+    }
```

Comment:
We should have access to mustache here, right? Should be a lot easier and more concise via mustache.

Reply:
At first, I also tried Mustache. However, the amount of event listeners and the variables that have to be set make it confusing again with Mustache and lead to a mixture of both worlds, which was somehow unattractive.

Reply:
I count 2